tests/testthat/test-tar_renv.R

```diff
@@ -1,28 +1,34 @@
 tar_test("tar_renv() works", {
```

Comment:
Would you also test the following?

* packages set with `tar_target(packages = ...)`
* packages from formats set with `tar_target(format = ...)`
* packages from a format set with `tar_option_set()`
* non-default `extras`